src/sgame/sg_bot_ai.cpp

```diff
@@ -1033,6 +1043,14 @@ AINodeStatus_t BotActionFight( gentity_t *self, AIGenericNode_t *node )
 	// We are human and we either are at fire range, or have
 	// a direct path to goal
 
+	glm::vec3 ownPos = VEC2GLM( self->s.origin );
+	glm::vec3 targetPos = mind->goal.getPos();
+	if ( ownPos.z < targetPos.z + 400 )
+	{
+		// activate the jetpack if we have it, but do not fly too high above the enemy
+		BotActivateJetpack( self );
```

Comment:
Does this mean the bot will start flying almost every time there is an enemy near?

Reply:
No. It means the bot will start flying if it can see the enemy.